bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -7,6 +7,30 @@
 	<bridge-type id="bridge">
 		<label>aWATTar Bridge</label>
 		<description>Provides price data from the aWATTar API.</description>
+
+		<channels>
+			<channel id="market-net" typeId="uom-price">
+				<label>Net Market Price</label>
+				<description>Price without VAT and network charge</description>
+			</channel>
+			<channel id="market-gross" typeId="uom-price">
+				<label>Gross Market Price</label>
+				<description>Price with VAT but without network charge</description>
+			</channel>
+			<channel id="total-net" typeId="uom-price">
+				<label>Net Total Price</label>
+				<description>Price with network charge but without VAT</description>
+			</channel>
+			<channel id="total-gross" typeId="uom-price">
+				<label>Gross Total Price</label>
+				<description>Price with network charge and VAT</description>
+			</channel>
```

Comment:
Have you considered dropping the gross channels and instead refer to the [VAT transformation](https://www.openhab.org/addons/transformations/vat/) in the documentation? I'm aware that the `vatPercent` can already be configured on the bridge, and that existing channels use it, but the question is if this is something to keep or eventually drop? Cc @Wolfgang1966 for comment. See also #14524.

Reply:
For the new TimeSeries functionality I think this makes sense to leave the gross channels away. For the already existing channels I recommend to mark VAT handling as deprecated and remove it in a later release, so users have some time to adapt their setup.

Reply:
Done

Reply:
Did you forget to push the commit?